### ISSUE TITLE
Add compatibility with xdebug to Docker container

### ISSUE THREAD

**Issue by e-caste** [WEEE-Open/WEEEhire-ng#53]
Find a way to enable debugging through the Docker container defined in the Dockerfile.  
@lvps you may want to chime in on this one to give additional details

**Comment by lvps** [comment]
@e-caste Yes: install xdebug (it's a php extension) and enable it. It should be enough... If it can connect to a server outside the container (the machine running the container, acting as a server).

Also find a way to avoid installing it at all in production

**Comment by lvps** [comment]
The container has to be split (nginx + PHP) to install xdebug, do the same thing as tarallo
